packages/shared/src/components/squads/SquadMemberItemAdditionalContent.tsx

```diff
@@ -74,6 +74,12 @@ function SquadMemberItemAdditionalContent({
       />
     </SimpleTooltip>
   );
+  const emptyOption = (
```

Comment:
Ah my initial thoughts where to simply give MR-{amount} to the element when it was loggedUser?
Personally not a big fan of rendering things for padding purposes 🙈

Reply:
Was just writing comment about that but you were too fast to review. I think this implementation with Button hidden is less error prone vs margin because if button styles change we get misalignment. WDYT?

Reply:
No strong feelings about it though, UIs by themself are error prone in the nature 😆

Reply:
I think the number of items on the right side will be constant as we can keep everything under the options menu. So margin would probably be an alright solution for this case.

Reply:
UIs are meant to be broken 😂

Reply:
Ah I see where you coming from.
Yeah this type of UI rather gets broken I think.

The issue with this is it might become unclear for people why it's there and cause confusion down the line.
But something to be said for both 😂